server/remote/bitbucket/fixtures/handler.go

```diff
@@ -42,6 +43,11 @@ func Handler() http.Handler {
 	return e
 }
 
+func getWorkspace(c *gin.Context) {
+	// role, _ := c.Params.Get("role")
```

Comment:
this is needed to let tests pass again (mock get workspaces call)

Reply:
(I mean the todo below has to be solved)

Reply:
Solved to the best of my little go knowledge. Mostly adapted the old getUserTeams() method and returning a correct workspaces object. Mostly unsure about the switch clause and whether the role should be used. At least tests are passing but no idea if the tests are correct now.